packages/publish-views/lib/shacl.js

```diff
@@ -0,0 +1,44 @@
+import through2 from 'through2'
+import { sh } from '@tpluscode/rdf-ns-builders'
+import { ValidationError } from './ValidationError.js'
+
+const REPORTS_KEY = 'COMBINED_SHACL_REPORTS'
+
+export function combineShaclReports({ context, report }) {
+  const variables = context.variables.has(REPORTS_KEY)
+    ? context.variables
+    : context.variables.set(REPORTS_KEY, [])
+
+  variables.get(REPORTS_KEY).push(report)
+
+  // continue stream
+  return true
+}
```

Comment:
A callback to `barnard59-validate-shacl` to continue processing all of the views.

It will only kill the pipeline when everything is done (see below)

Reply:
Interesting functionality, I wonder if this should go as a functionality of `barnard59-validate-shacl` itself, what do you think?

Reply:
Maybe. I did something a little similar in cube creator too because I also did not want the pipeline to be stopped on first error

Reply:
I think it's a very reasonable functionality.  

I'm unsure about one REPORTS_KEY if there are many validator instances in the pipeline though